Assets/Scripts/Enemies/EnemyMovement.cs

```diff
@@ -3,12 +3,15 @@
 public abstract class EnemyMovement : MonoBehaviour
 {
     EnemyController EnemyController;
+    protected Vector3 up;
+    public float speed;
 
     void Start()
     {
         var healthbar = FindGameObject<HealthBar>("health");
         var spawner = FindGameObject<RandomSpawner>("spawner");
         EnemyController = new EnemyController(this, spawner, healthbar);
+        up = transform.up;
```

Comment:
I didn't realize that `Vector3.up` was not the same as `transform.up`. Either way the correct up is alread available through the `EnemyPosition` https://github.com/fullerga/Dodge/blob/NewLevelsPullRequest/Assets/Scripts/Enemies/EnemyPosition.cs#L13